crates/ruff_linter/src/rules/pep8_naming/rules/camelcase_imported_as_acronym.rs

```diff
@@ -23,6 +22,10 @@ use crate::rules::pep8_naming::settings::IgnoreNames;
 /// Note that this rule is distinct from `camelcase-imported-as-constant`
 /// to accommodate selective enforcement.
 ///
+/// Also note that import aliases following an import convention according to the
+/// [`lint.flake8-boolean-trap.extend-allowed-calls`] option are allowed.
+///
+///
```

Comment:
```suggestion
/// [`lint.flake8-import-conventions.banned-aliases`] option are allowed.
///
```

Also, I notice you used square brackets but didn't add a link at the end. Was that deliberate -- do we have a mechanism that adds links automatically to settings?
```

Reply:
Fancy script or no fancy script, you're still referring to the wrong setting here: the `lint.flake8-boolean-trap.extend-allowed-calls` setting has nothing to do with this rule

Reply:
I filed #12935 to address this